### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by luciaL** [Jia-Research-Lab/PFENet#22]
hello, I wanna try few-shot segmentation for medical images. There are two questions I could not understand. I hope you can help me.
1. The ground truth MQ of the query image is invisible to the model. So how did you use cross entropy loss for training?
2. In experiments part, what does 1-shot and 5-shot mean in Table 2? Could you please give me more implementation details?

**Comment by tianzhuotao** [comment]
@luciaL 

Unfortunately, if the ground truth images are missing, our method might not be applicable to your setting. 

In semi- and weakly- supervised segmentation tasks, they use CAM to retrieve weak pixel-level annotations for those image-level labeled images, perhaps you can take a reference from their works.

1-shot and 5-shot mean during testing there are 1 and 5 support samples respectively for each task.
